### ISSUE TITLE
Relative paths in tests fail in CI

### ISSUE THREAD

**Issue by aaron-wilson-bfg** [bigfishgames/GameBenchAPI#2]
The relative path created in the integration tests works in PyCharm but fail in Travis CI. See the log:

```bash
ERROR: test_set_data (gamebench_api_client.tests.test_integration.test_api.test_response.test_response_mediator.TestGenericFrameMediator)
Verify get_data returns a DataFrame.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/bigfishgames/GameBenchAPI/gamebench_api_client/tests/test_integration/test_api/test_response/test_response_mediator.py", line 96, in setUp
    PARENT_DIR + API_SAMPLES + "sessionid.json")) as json_data:
FileNotFoundError: [Errno 2] No such file or directory: '/home/travis/build/bigfishgames/GameBenchAPI/gamebench_api_client/tests/fixtures/API_SAMPLES/sessionid.json'
```
You can see the path for the fixture is missing some directories.

**Comment by aaron-wilson-bfg** [comment]
#4 Is blocked. Beginning work to update how the paths are generated.

**Comment by aaron-wilson-bfg** [comment]
Fixed and in #4
